sphera/src/components/Table/DeleteDialaog.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+function DeleteDialaog(someParameter, deleteEl, setDeleteEl) {
```

Comment:
Change to props

Reply:
This file is unnecessary, have no use, I removed it.